packages/js/product-editor/src/blocks/notice/block.json

```diff
@@ -0,0 +1,42 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "woocommerce/product-notice",
```

Comment:
I would update the name to make it more `variations` specific, since we still have the `hasAttributesUsedForVariations( productAttributes )` in place and only show the notice if that is true.

Maybe `woocommerce/product-has-variations-notice`?